account_easy_reconcile/base_reconciliation.py

```diff
@@ -93,27 +98,28 @@ def _where(self, rec, *args, **kwargs):
         # but as we use _where_calc in _get_filter
         # which returns a list, we have to
         # accomodate with that
-        params = [rec.account_id.id]
-        if rec.partner_ids:
+        params = [self.account_id.id]
+        if self.partner_ids:
             where += " AND account_move_line.partner_id IN %s"
-            params.append(tuple([l.id for l in rec.partner_ids]))
+            params.append(tuple([l.id for l in self.partner_ids]))
         return where, params
 
-    def _get_filter(self, cr, uid, rec, context):
-        ml_obj = self.pool.get('account.move.line')
+    @api.multi
+    def _get_filter(self):
+        ml_obj = self.env['account.move.line']
         where = ''
         params = []
-        if rec.filter:
+        if self.filter:
             dummy, where, params = ml_obj._where_calc(
-                cr, uid, eval(rec.filter), context=context).get_sql()
+                eval(self.filter)).get_sql()
```

Comment:
please change this to use safe_eval. 

A PR on 7.0 to fix this is welcome too.